Lib9c/Action/RankingBattle.cs

```diff
@@ -192,12 +193,15 @@ public override IAccountStateDelta Execute(IActionContext context)
 
             ArenaInfo = new ArenaInfo((Dictionary)weeklyArenaMap[arenaKey]);
             EnemyArenaInfo = new ArenaInfo((Dictionary)weeklyArenaMap[enemyKey]);
+            var rankingSheets = states.GetRankingSimulatorSheets();
+            var player = new Player(avatarState, rankingSheets);
+            var enemyPlayer = new EnemyPlayer(enemyAvatarState, rankingSheets);
             var simulator = new RankingSimulator(
                 ctx.Random,
-                avatarState,
-                enemyAvatarState,
+                player,
+                enemyPlayer,
```

Comment:
I guess it would be better it takes `SimualationEnemyPlayer` than `EnemyPlayer` because `SimualationEnemyPlayer` is smaller than `EnemyPlayer`.